airflow/jobs/scheduler_job.py

```diff
@@ -798,7 +798,7 @@ def _update_dag_run_state_for_paused_dags(self, session: Session = NEW_SESSION)
                 )
                 for dag_run in dag_runs:
                     dag_run.dag = dag
-                    _, callback_to_run = dag_run.update_state(execute_callbacks=False)
+                    dag_run, callback_to_run = dag_run.update_state(execute_callbacks=False)
```

Comment:
I think 
```suggestion
                    _, callback_to_run = dag_run.update_state(execute_callbacks=False)
```

given this function is defined as `tuple[list[TI], DagCallbackRequest | None]`

(I'm surprised this passes tests _at all_!!)

Reply:
Oh I guess dag_run isn't used after this call

Reply:
I've used it before, removed in the latest PR version and this leftover stayed. Fixed.

BTW, I think it's not detected by static checks because it's used in the same block before anyway.